### DISCUSSION TITLE
Restore support for Android 8

### DISCUSSION THREAD

**Discussion by muntasimulhaque** [standardnotes/forum#3011] [Suggestions]
This morning when I opened Standard Notes on my Vivo Y90 Smartphone (comes with customized Android 8.1), it showed a all-white screen. No UI was visible except this all-white background. But Android UI was visible and working. Only app area showed the white background.

It happened every time I tried to open Standard Notes.

I then uninstalled the app and tried to reinstall it from Google Play Store. But I couldn't find it there. Only got the following message:

![WhatsApp Image 2023-04-09 at 18 06 20](https://user-images.githubusercontent.com/106990977/230774861-f8afc4a0-1f6d-41be-84fd-adbdce7700f7.jpeg)

Did Standard Notes stop supporting Android 8.1? If yes, then why? It's not that older. My life was centered around Standard Notes. What will I do now?

**Comment by ghost** [comment]
StandardNotes requires sdk 28 at least, i.e. android 9 : https://apilevels.com/

**Comment by muntasimulhaque** [comment]
> StandardNotes requires sdk 28 at least, i.e. android 9 : https://apilevels.com/

Why drop support for Android 8.1? What would we do?

**Comment by ghost** [comment]
> > StandardNotes requires sdk 28 at least, i.e. android 9 : https://apilevels.com/
> 
> Why drop support for Android 8.1? What would we do?

I don't know if it's a security reason or a functionality reason. But android 8 is no longer maintained, as android 9, so you should consider upgrading to a recent version of android anyway.

**Comment by muntasimulhaque** [comment]
> 

Unfortunately, my phone's model doesn't support Android version higher than 8.1. 

So, those who got unlucky like me, should we abandon Standard Notes? I have so many notes on it. And cross platform syncing is important to me. 

To the dev: please see whether you can bring back support for Android version 8.1. It'd really mean much to me and all those who are in the same situation after this change.

**Comment by ghost** [comment]
> > 
> 
> Unfortunately, my phone's model doesn't support Android version higher than 8.1.
> 
> So, those who got unlucky like me, should we abandon Standard Notes? I have so many notes on it. And cross platform syncing is important to me.
> 
> To the dev: please see whether you can bring back support for Android version 8.1. It'd really mean much to me and all those who are in the same situation after this change.

You can use standard notes in PWA in the meantime, I think it should work through your browser.

**Reply by cjholly** [reply]
Anybody have any work arounds besides the web app? What happens if I keep using compatible version on my Gemini Planet which is Android 8.1? How will that affect syncing with other devices that use the current app?
